gulp-tasks/utils/build-browser-bundle.js

```diff
@@ -36,6 +36,8 @@ const globals = (moduleId) => {
   }
 
   const packageName = splitModuleId.shift();
+
+  // TODO(philipwalton): remove this restriction.
```

Comment:
This needs a little nuance other than just remove. If this is removed, how do you plan on linking the browser namespace to the file path in the ES2015 import?

Reply:
Updated the comment based on our discussion to add future support for importing things like `workbox-routing/Route.mjs`